### DISCUSSION TITLE
Windows 11 ARM64 discussions

### DISCUSSION THREAD

**Discussion by darren2046** [WhatTheBlock/WindowsSimplify#49] [General]
如果您感兴趣的话我这里有iso，在parallels desktop这里它自动下载的，我可以分享给你。

22000.318.211104-1236.co_release_svc_refresh_CLIENTCONSUMER_RET_A64FRE_en-us.iso

**Comment by WhatTheBlock** [comment]
我以前有製作，但我沒有ARM的裝置可以測試

[link](https://github.com/WhatTheBlock/WindowsSimplify/releases/tag/w11.220123)

**Comment by darren2046** [comment]
<img width="1092" alt="image" src="https://user-images.githubusercontent.com/107549747/178560640-41f2f6c3-1bf4-4384-ac3b-7b762953cd89.png">

我测试了一下从iso启动，蓝屏。我再看看怎样装。

**Comment by darren2046** [comment]
我发现我没有ARM的WindowsPE。就尴尬。

**Comment by WhatTheBlock** [comment]
感謝測試，請問這是啟動windows setup就藍屏嗎？還是安裝後才這樣呢？如果是前者那才有可能利用winpe規避安裝過程的錯誤

**Comment by darren2046** [comment]
是啟動windows setup就藍屏诶

**Comment by WhatTheBlock** [comment]
好的，我知道了

下次更新arm版的時候再通知你幫忙測試

謝謝

**Comment by darren2046** [comment]
好的。谢谢大哥。

**Comment by WhatTheBlock** [comment]
你好，幫我試試此版本是否能安裝
[Link](https://gmnfuedutw-my.sharepoint.com/:u:/g/personal/40543229_gm_nfu_edu_tw/EeGy0pfHeCRAsICjV7JyB5cBgXQ3WVu_q8Mh1obyqVISfA?e=2wPkfF)

**Comment by darren2046** [comment]
可以安装。

<img width="1092" alt="image" src="https://user-images.githubusercontent.com/107549747/184103195-03451959-0ddb-4444-a195-dd81bd0a7fe2.png">

扩展程序也装上了。

但是它说它需要更新。

<img width="1076" alt="image" src="https://user-images.githubusercontent.com/107549747/184104134-90c17d06-4345-4587-bb43-2d13cb1f64e7.png">

不过好像用着没什么问题，这个模式也可以用。

<img width="1670" alt="image" src="https://user-images.githubusercontent.com/107549747/184105456-3bfd48db-529a-481d-9de3-fb2ef9eeb9b6.png">

**Comment by darren2046** [comment]
notepad和terminal测试是打不开

<img width="1120" alt="image" src="https://user-images.githubusercontent.com/107549747/184106743-85d5fd81-5bf3-4657-9a21-fdc2947c89e5.png">

而且外部复制的文本在虚拟机里面不能粘贴。

**Comment by WhatTheBlock** [comment]
> notepad和terminal测试是打不开

試試這個指令，package要改成你無法開啟的app名稱
`Reset-AppxPackage -Package Microsoft.WindowsTerminal_1.12.10983.0_x64__8wekyb3d8bbwe`

取得package list
`Get-AppxPackage | Select PackageFullName`

> 而且外部复制的文本在虚拟机里面不能粘贴。

這個需要虛擬機的驅動和系統相容才行，我也沒辦法除錯

**Comment by WhatTheBlock** [comment]
方便的話想請你幫忙測試這個版本
我使用不同的方式精簡

[Link](https://gmnfuedutw-my.sharepoint.com/:u:/g/personal/40543229_gm_nfu_edu_tw/EctSfjl2EVNIvkZkMDd3CGYBn7vmImMiOA9JLtidf0ee3A?e=qGUCtK)

**Comment by oOJohn6Oo** [comment]
我尝试在Mac mini M1上使用[UTM](https://mac.getutm.app)安装[这个版本](https://github.com/WhatTheBlock/WindowsSimplify/releases/tag/w11.220505)的 Windows11，一直启动不成功。

诸位有安装成功的案例嘛？
<img width="912" alt="image" src="https://user-images.githubusercontent.com/24718357/197345441-817ee96d-663b-44c3-bd4d-f9713c11c0ec.png">

**Reply by WhatTheBlock** [reply]
Apple Silicon只能使用ARM64的版本
請至ARM64頁面下載使用

**Reply by oOJohn6Oo** [reply]
看UTM说明上，MAC无法直接安装22H2，所以我在 release 找了个能支持直接安装的最新的iso，没注意架构😂

**Reply by WhatTheBlock** [reply]
我製作了一個win10 arm64，你有空可以試試效果如何
嘗試製作win10是想確認在arm上是否一樣是win10對硬體要求較低

[17763.3534_arm_221025](https://github.com/WhatTheBlock/WindowsSimplify/releases/tag/w10.arm.221025)

**Reply by oOJohn6Oo** [reply]
测试未成功，下班再看看
<img width="372" alt="image" src="https://user-images.githubusercontent.com/24718357/197925050-6ace34df-f828-4ef1-ba3e-ac85309147ac.png">

**Reply by WhatTheBlock** [reply]
我找時間製作一個21h2的win11給你測試

**Reply by WhatTheBlock** [reply]
[22000.1165_arm_221102](https://github.com/WhatTheBlock/WindowsSimplify/releases/tag/w11.arm.221102)

**Reply by oOJohn6Oo** [reply]
请问这个怎么安装？下载完合成为ISO？

**Reply by WhatTheBlock** [reply]
這只是分割壓縮檔，解壓縮後就是ISO

**Reply by oOJohn6Oo** [reply]
这个是22H2，安装不了
<img width="912" alt="image" src="https://user-images.githubusercontent.com/24718357/199689466-df8773d9-c92f-4e4e-be8f-5c1ab1381c3d.png">

**Reply by WhatTheBlock** [reply]
22000是21H2
2262X才是22H2
你要檢查一下虛擬機有沒有開TPM模擬
或是記憶體太小導致沒有達到微軟要求的門檻

**Comment by carlchina** [comment]
<img width="702" alt="image" src="https://user-images.githubusercontent.com/5435458/205793895-6e367751-3554-4f1c-8e47-e1ecb97d6c25.png">
win11 arm 22621 无法下载语言包，尝试多次都失败了。

**Reply by WhatTheBlock** [reply]
這是高度精簡版本，所以不支持安裝語言包

**Reply by carlchina** [reply]
感谢，明白了。

**Comment by rongpengli** [comment]
精简版本无法安装Office，这个可以解决吗？

**Reply by WhatTheBlock** [reply]
可以提供錯誤畫面嗎

**Comment by fwycxm** [comment]
<img width="1030" alt="截屏2023-09-14 18 18 43" src="https://github.com/WhatTheBlock/WindowsSimplify/assets/126217568/4692b578-f339-451d-9a81-871670a87a74">
